swap/src/protocol/bob/state.rs

```diff
@@ -35,8 +35,8 @@ pub enum BobState {
     CancelTimelockExpired(State4),
     BtcCancelled(State4),
     BtcRefunded(State4),
-    XmrRedeemed,
-    BtcPunished,
+    XmrRedeemed(State6),
```

Comment:
I don't think we should define a `State6`. Let me try something locally see if it looks better.

Reply:
@da-kami https://github.com/comit-network/xmr-btc-swap/pull/146